cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -87,6 +87,19 @@ func (p *Provisioner) ProvisionHostPath(ctx context.Context, opts pvController.P
 		return nil, pvController.ProvisioningFinished, iErr
 	}
 
+	bsoft := opts.StorageClass.Parameters["bsoft"]
+	bhard := opts.StorageClass.Parameters["bhard"]
+
+	if bsoft != "" ||
+		bhard != "" {
+
+		qErr := p.createInitQuotaPod(ctx, podOpts, bsoft, bhard)
+		if qErr != nil {
+			klog.Infof("Setting quota failed: %v", name, qErr)
+			return nil, pvController.ProvisioningFinished, qErr
+		}
+	}
```

Comment:
Add logs to indicate the success scenario also.

Reply:
Added logs